dingo/pipe/asimov.py

```diff
@@ -0,0 +1,319 @@
+# adapted from the Asimov Bilby Pipeline interface
+import configparser
+import glob
+import os
+import re
+import subprocess
+import time
+
+
+from asimov import config, logger
+
+from asimov.pipeline import Pipeline, PipelineException, PipelineLogger, PESummaryPipeline
```

Comment:
Yes, looks like it will be required.

Reply:
I don't know if that's a good idea, since Asimov requires htcondor..

Reply:
Hmm. Do the packages inherit this dependency? If they are built for the LSC, it should be included. If asimov is fairly lightweight to install, it should not impact installations outside of the LSC.
htcondor is not a Python package, so it can't be installed via `pyproject.toml` anyway.

Reply:
I could only install Asimov (and dingo with asimov) on the cluster where condor was installed, but whatever you think is the best

Reply:
Ok, what I said above is incorrect, there is indeed a `htcondor` Python package (the Python bindings for HTCondor).
It won't work without having HTCondor on the system, however I was able to `pip install asimov` in my dingo environment.

Reply:
I think the solution is to not make Asimov a dependency. That module will never get loaded by anything other than Asimov anyway.

Reply:
ah really? Then they've updated that

Reply:
Ok!

Reply:
Ah, if you think it can be installed without htcondor then go ahead and add as a dependency. But only if it works; like I said this module will only ever get used **by** Asimov.

Reply:
Well, if it didn't work in the past then I'm reluctant to recommend that it should be a dependency.